kafka-hello-world/src/main/scala/com/labs1904/hwe/homework/theory.md

```diff
@@ -0,0 +1,48 @@
+# Overview
```

Comment:
Lets move this into the Kafka README and add one step at the end to understand KafkaHomework.scala and attempt to hook it up to our cluster.